flang/lib/Optimizer/CodeGen/CodeGen.cpp

```diff
@@ -1141,9 +1185,26 @@ struct EmboxCommonConversion : public FIROpConversion<OP> {
     dest =
         insertField(rewriter, loc, dest, {5},
                     this->genConstantOffset(loc, rewriter, getCFIAttr(boxTy)));
-    dest = insertField(
-        rewriter, loc, dest, {6},
-        this->genConstantOffset(loc, rewriter, isDerivedType(boxTy)));
+    const bool hasAddendum = isDerivedType(boxTy);
+    dest = insertField(rewriter, loc, dest, {6},
+                       this->genConstantOffset(loc, rewriter, hasAddendum));
+
+    if (hasAddendum) {
+      // TODO: For now, set it to zero. The runtime expects some information
```

Comment:
I think `it` refers to the extra fields bit flags, non?

Reply:
Oui. Replaced by `addendum flags`.